ChallengeTestRunner/test/test_challenge_test.py

```diff
@@ -237,7 +237,7 @@ def test_write(self):
 '''
 
 
-class ChallengeTestTest(unittest.TestCase):
+class ChallengeTestTest(challenge_test.TestCaseWrapper):
```

Comment:
This should be unittest.TestCase, not challenge_test.TestCaseWrapper. We are testing the TestCaseWrapper class, so we need to use an external library to test it.